To: vim_dev@googlegroups.com
Subject: Patch 7.3.069
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.069
Problem:    GTK: pressing Enter in inputdialog() doesn't work like clicking OK
	    as documented.
Solution:   call gtk_entry_set_activates_default(). (Britton Kerin)
Files:	    src/gui_gtk.c


*** ../vim-7.3.068/src/gui_gtk.c	2010-08-15 21:57:32.000000000 +0200
--- src/gui_gtk.c	2010-11-24 18:44:21.000000000 +0100
***************
*** 1287,1292 ****
--- 1287,1295 ----
  	entry = gtk_entry_new();
  	gtk_widget_show(entry);
  
+ 	/* Make Enter work like pressing OK. */
+         gtk_entry_set_activates_default(GTK_ENTRY(entry), TRUE);
+ 
  	text = CONVERT_TO_UTF8(textfield);
  	gtk_entry_set_text(GTK_ENTRY(entry), (const char *)text);
  	CONVERT_TO_UTF8_FREE(text);
*** ../vim-7.3.068/src/version.c	2010-11-24 17:59:27.000000000 +0100
--- src/version.c	2010-11-24 18:46:39.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     69,
  /**/

-- 
Why I like vim:
> I like VIM because, when I ask a question in this newsgroup, I get a
> one-line answer.  With xemacs, I get a 1Kb lisp script with bugs in it ;-)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
